medusa/service/grpc/server.py

```diff
@@ -44,11 +46,14 @@ def __init__(self, config):
         self.storage = Storage(config=self.config.storage)
 
     def Backup(self, request, context):
-        logging.info("Performing backup {}".format(request.name))
+        logging.info("Performing backup {} (type={})".format(request.name, request.mode))
         resp = medusa_pb2.BackupResponse()
-        # TODO pass the staggered and mode args
+        # TODO pass the staggered arg
+        mode = BACKUP_MODE_DIFFERENTIAL
+        if medusa_pb2.Mode.FULL == request.mode:
```

Comment:
@MattFellows I missed this on my initial review. `medusa_pb2.Mode.FULL` is wrong. It should be `medusa_pb2.BackupRequest.Mode.FULL`. It is causing some integration tests to fail.

Reply:
There are 3 ways of referencing that enum that seem to pass local testing - you could of just told me that the 3rd was was wrong  (always nice to get a Monty Hall Problem reference in ;) )